<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="artists">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>Artist</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="artist"/>
             </item>
             <item>
              <widget class="QPushButton" name="addArtist">
               <property name="text">
                <string>&amp;Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeArtist">
               <property name="text">
                <string>&amp;Remove</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="albums">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>Year</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Album</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Borrowed</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="year"/>
             </item>
             <item>
              <widget class="QLineEdit" name="title"/>
             </item>
             <item>
              <widget class="QCheckBox" name="borrowed">
               <property name="text">
                <string>Borrowed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="addAlbum">
               <property name="text">
                <string>&amp;Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeAlbum">
               <property name="text">
                <string>&amp;Remove</string>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>widget_2</zorder>
            <zorder>year</zorder>
            <zorder>addAlbum</zorder>
            <zorder>removeAlbum</zorder>
            <zorder>title</zorder>
            <zorder>borrowed</zorder>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="logs"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>DB type</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="radioButton">
               <property name="text">
                <string>MongoDB</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_2">
               <property name="text">
                <string>CouchDB</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_3">
               <property name="text">
                <string>Redis</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear">
            <property name="text">
             <string>&amp;Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fill">
            <property name="text">
             <string>&amp;Fill</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
